frontend/storybook/src/components/entry/homepage/footer/footer-jigs.ts

```diff
@@ -0,0 +1,35 @@
+import "@elements/entry/home/TOSORT/column-details";
+import "@elements/entry/home/TOSORT/column-list";
+
+export default {
+    title: 'Homepage Paragraph',
+  }
+  const STR_TITLE="JIGs";
+  const STR_LINE1="Hebrew JIGs";
+  const STR_LINE2="Jewish holidays JIGs";
+  const STR_LINE3="Torah JIGs";
+  const STR_LINE4="J-Stream";
+  const STR_LINE5="Israel";
+  const STR_LINE6="Songs";
+  const STR_WHITE ="white";
+
+export const footerjigs= () => {
+    return `
+    <column-details head_title="${STR_TITLE}">
+    <column-list text_line="${STR_LINE1}" color="${STR_WHITE}" slot="list" ></column-list>
```

Comment:
The static content should be in elements, not the component.

Reply:
This has been fixed